spec/datadog/core/telemetry/v1/telemetry_request_spec.rb

```diff
@@ -38,9 +38,9 @@
     )
   end
   let(:request_type) { 'app-started' }
-  let(:runtime_id) { '20338dfd-f700-0d470f054ae8' }
+  let(:dummy_runtime_id) { '20338dfd-f700-0d470f054ae8' }
   let(:seq_id) { 42 }
-  let(:session_id) { '20338dfd-f700-0d470f054ae8' }
+  let(:dummy_session_id) { '20338dfd-f700-0d470f054ae8' }
   let(:tracer_time) { 1654805621 }
```

Comment:
Ah, apologies for not being clearer on my previous comment. I actually was talking about the value of the let, e.g.

```ruby
let(:runtime_id) { 'dummy_runtime_id' }
```

...not actually the name. (But it's also ok to use `let(:dummy_runtime_id) { 'dummy_runtime_id') `.

My concern is that I don't like using that data that looks real but isn't, because it can leave people thinking -- wait, is there another format other than UUID v4 that we need to support?

Reply:
This makes a lot of sense - changed!